cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -274,6 +286,91 @@ public void upgradeClusterRuntime(Set<ClusterComponent> components, boolean patc
         }
     }
 
+    private void tagHostsWithHostTemplateName() throws ApiException, CloudbreakException {
+        ClouderaManagerRepo clouderaManagerRepoDetails = clusterComponentProvider.getClouderaManagerRepoDetails(stack.getCluster().getId());
+        if (!isVersionNewerOrEqualThanLimited(clouderaManagerRepoDetails::getVersion, CLOUDERAMANAGER_VERSION_7_6_0)) {
```

Comment:
could you invert this if, add the logic here and add an else branch with the log?